src/System.Drawing.Common/tests/BufferedGraphicsContextTests.cs

```diff
@@ -254,9 +254,11 @@ public void Finalize_Invoke_Success()
         {
             // Don't allocate anything as this would leak memory.
```

Comment:
I know you didn't write this, but this first sentence makes no sense to me :)

Reply:
Fixed.